lib/Client.js

```diff
@@ -604,14 +602,14 @@ class Client extends EventEmitter {
     * @arg {Boolean} [options.hoist] Whether to hoist the role in the user list or not
     * @arg {Boolean} [options.mentionable] Whether the role is mentionable or not
     * @arg {String} [options.name] The name of the role
-    * @arg {Number} [options.permissions] The role permissions number
+    * @arg {Number | Permission} [options.permissions] The role permissions
```

Comment:
We can allow string here
```suggestion
    * @arg {Number | String | Permission} [options.permissions] The role permissions
```